bitrise.yml

```diff
@@ -27,7 +26,7 @@ workflows:
 
   create-release-version:
     envs:
-    - RELEASE_VERSION: 0.1.2
+    - RELEASE_VERSION: 1.0.0
```

Comment:
Wasn't quite sure if I should bump this to 1.0.0 since we're removing a feature that some people might be using.